backend/routers/buildings.js

```diff
@@ -22,10 +22,31 @@ const myBuildings = {
   ],
 };
 
+const mockedOneBuilding = {
+  id: 2,
+  type: 'farm',
+  level: 1,
+  hp: 1,
+  started_at: 12345789,
+  finished_at: 12399999,
+};
+
 function getBuildings(req, res) {
   return res.status(200).send(myBuildings);
 }
 
+function getBuildingById(req, res) {
+  const buildingId = Number.parseInt(req.params.buildingId, 10);
+  if (Number.isNaN(buildingId) || buildingId < 0) {
+    return res.sendStatus(400);
+  }
+  if (buildingId === 0) {
+    return res.sendStatus(404);
+  }
```

Comment:
no need to rewrite this, I just have a question: why 0 is handled differently than -1 or -2? 
why is there a 400 and a 404 status codes? 
just being curious.